modules/fluentd/manifests/init.pp

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class fluentd (
+    String $worker_type,
+    String $stackdriver_project,
+) {
+
+    $stackdriver_keyid    = lookup("stackdriver.${stackdriver_project}.keyid")
```

Comment:
I added these secrets to the vault.yaml on install2 mdc1/2

Reply:
As we discussed over video, I'd like to see these lookups done from a profile and passed as parameters to the fluentd class.  A generic 'logging' profile should be used for all roles.